pkg/printutil/printutil.go

```diff
@@ -35,6 +35,18 @@ type Table struct {
 	// Function which will eval whether to apply color to a col
 	// ColorColCond    func(t *Table) (bool, error)
 	// ColorColCode [2]string
+
+	altPadding []int
+
+	headerPadding []int
```

Comment:
Are we able to re-use altPadding here? Is there a usecase when the header wouldn't be the same to the rest of the columns if we're left aligning?

Reply:
Removed headerPadding and associated methods in favor of altPadding.